test/basic.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+The complete set of authors may be found at http://polymer.github.io/AUTHORS
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+-->
+<html id="html">
+<head>
+  <meta charset="UTF-8">
+  <title>iron-scroll-threshold test</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+
+  <link rel="import" href="../iron-scroll-threshold.html">
+</head>
+<body>
+
+  <test-fixture id="trivialScrollTheshold">
+    <template>
+      <iron-scroll-theshold></iron-scroll-theshold>
+    </template>
+  </test-fixture>
+
+<script>
+
+  suite('basic features', function() {
+    var scrollableElement;
+
+    setup(function() {
+      scrollableElement = fixture('trivialScrollTheshold');
+    });
+
+    test('default', function() {
+
```

Comment:
I think this PR is ok to merge now, but did you want to add any functional test here?  Looks like it's just stubbed out...